concepts/privacy.mdx

```diff
@@ -1,21 +1,25 @@
 
-## Provider Privacy
+### Anonymous Receiver-Updatable Broadcast Encryption (ARUBE)
 
-A provider who receives a bid can make a commitment to it, which they also sign. This action binds them to the bid. Furthermore, the process is executed in such a way that only the bidder who issued the bid can identify a commitment as being related to their bid. This approach ensures a high level of privacy for the providers, as the commitments they issue are not associated with a specific bid throughout the auction round.
+ARUBE is the primitive that allows bidders in the mev-commit protocol to privately send bids to a subset of the providers without revealing what this subset is. This functionality is commonly provided by anonymous broadcast encryption [[BBW06]](https://www.cs.utexas.edu/~bwaters/publications/papers/privatebe.pdf). A feature of broadcast encryption is that a new set of recipients can be specified for each message that is encrypted. In our use-case, the set of recipients for a given bidder typically remains stable since bidders mostly send their bids to the set of providers they work with and only occasionally add or remove a provider from that set. To be able to leverage this for more efficient schemes, we refine the notion of broadcast encryption by adding an algorithm for updating the set of recipients, while the encryption algorithm encrypts a message for a previously determined recipient set. We call this notion anonymous receiver-updatable broadcast encryption (ARUBE).
 
-### Signature Schemes
 
-We implement ECDSA (Elliptic Curve Digital Signature Algorithm) keys for our signature process.
-Additionally, our messages are formatted using EIP-712 standards, enhancing human readability and structured data representation.
-The ECDSA framework enables the recovery of both the public key and the signer's address through the ECRecover function, which are native to Geth.
+## Providers
+
+A provider who receives a bid can make a commitment to it, which they also sign for authentication. The signed commitment binds the provider to the bid. Furthermore, the process is executed in such a way that only the bidder who issued the bid can identify a commitment as being related to their bid. This approach ensures a high level of privacy for the providers, as the commitments they issue are not linkable to a specific bid throughout the auction round. Moreover, if the provider refuses to open a commtiment at the end of the round, the bidder can independently open the commitment without needing assistance from the provider. This functionality is provided by the novel notion of dual-phase commitments.
```

Comment:
Before saying the bidder can also open, I'd add that usually the provider would open this after the L1 block has been built.